app/assets/stylesheets/components/_product_card.scss

```diff
@@ -44,7 +44,7 @@
       background-color: #FFFFFF;
       padding: 12px 20px;
       z-index: 1;
-      height: 40%;
+      height: 35%;
```

Comment:
height looks a little small now for the important info, perhaps play around with the height in graduatlly increasing this as you decrease the height of the total card